draft-ietf-quic-transport.md

```diff
@@ -3804,6 +3804,14 @@ number length, connection ID length, and path MTU.  A receiver MUST accept
 packets containing an outdated frame, such as a MAX_DATA frame carrying a
 smaller maximum data than one found in an older packet.
 
+A sender SHOULD avoid retransmitting information from packets once they are
+acknowledged. This includes packets that are acknowledged after being declared
+lost, which can happen in the presence of network reordering. Doing so requires
+senders to retain information about packets after they are declared lost. A
```

Comment:
```suggestion
acknowledged. This includes packets acknowledged after being
declared lost due to network reordering, which requires senders to
retain information about packets after they are declared lost. A
```

Reply:
That's a run-on sentence.

Reply:
Fair, I just find the current structure awkward.  Either way, this proposal is basically done.